src/LogMonitor.js

```diff
@@ -72,6 +72,10 @@ export default class LogMonitor extends Component {
 
   shouldComponentUpdate = shouldPureComponentUpdate;
 
+  state = {
```

Comment:
Can we use `monitorState` and `reducers` for that? We already have a reducer tracking current scroll top, for example. Local state will get erased on reload, but with `persistState()`, `monitorState` will not.